subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformUpstreamDependenciesResolverFactory.java

```diff
@@ -19,7 +19,7 @@
 import org.gradle.api.artifacts.component.ComponentIdentifier;
 
 /**
- * Factory for {@link TransformUpstreamDependenciesResolver} that relies on the information provided to its {@code create} method.
+ * ResolvedArtifactTransformer for {@link TransformUpstreamDependenciesResolver} that relies on the information provided to its {@code create} method.
```

Comment:
Watch out for automatic renames like this.i think this should be reverted and the PR should be checked for similar errors

Reply:
I thought I got all of these.  IntelliJ was very aggressive about this.  I'll make another pass to check for them.